UndertaleModTool/Converters/UndertaleCachedImageLoader.cs

```diff
@@ -300,7 +300,12 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
             if (textures is null)
                 return null;
 
-            int index = (int)(float)values[1];
+            int index = -1;
+            if (values[1] is int indexInt)
+                index = indexInt;
+            else if (values[1] is float indexFloat)
+                index = (int)indexFloat;
+
```

Comment:
Can't this be replaced with just
```cs
int index = (int)values[1];
```
?

Reply:
`values[1]` is `object`, so `(int)values[1]` here is _"unboxing"_, not a type conversion.
So no, it can't be replaced.